src/Entity/EventTranslation.php

```diff
@@ -4,66 +4,33 @@
 
 namespace App\Entity;
 
+use App\Repository\EventTranslationRepository;
+use Doctrine\DBAL\Types\Types;
 use Doctrine\ORM\Mapping as ORM;
 use Sulu\Component\Persistence\Model\AuditableInterface;
 use Sulu\Component\Persistence\Model\AuditableTrait;
 
-/**
- * @ORM\Entity(repositoryClass="App\Repository\EventTranslationRepository")
- */
+#[ORM\Entity(repositoryClass: EventTranslationRepository::class)]
 class EventTranslation implements AuditableInterface
 {
     use AuditableTrait;
 
-    /**
-     * @var int|null
-     *
-     * @ORM\Id()
-     * @ORM\GeneratedValue()
-     * @ORM\Column(type="integer")
-     */
-    private $id;
-
-    /**
-     * @var Event
-     *
-     * @ORM\ManyToOne(targetEntity="App\Entity\Event", inversedBy="translations")
-     * @ORM\JoinColumn(nullable=false)
-     */
-    private $event;
-
-    /**
-     * @var string
-     *
-     * @ORM\Column(type="string", length=5)
-     */
-    private $locale;
-
-    /**
-     * @var string|null
-     *
-     * @ORM\Column(type="string", length=255, nullable=true)
-     */
-    private $title;
-
-    /**
-     * @var string|null
-     *
-     * @ORM\Column(type="text", nullable=true)
-     */
-    private $teaser;
-
-    /**
-     * @var string|null
-     *
-     * @ORM\Column(type="text", nullable=true)
-     */
-    private $description;
-
-    public function __construct(Event $event, string $locale)
+    #[ORM\Id]
+    #[ORM\GeneratedValue]
+    #[ORM\Column(type: Types::INTEGER)]
+    private ?int $id = null;
+
+    #[ORM\Column(type: Types::STRING, length: 255, nullable: true)]
+    private ?string $title = null;
+
+    #[ORM\Column(type: Types::TEXT, nullable: true)]
+    private ?string $teaser = null;
+
+    #[ORM\Column(type: Types::TEXT, nullable: true)]
+    private ?string $description = null;
+
+    public function __construct(#[ORM\ManyToOne(targetEntity: Event::class, inversedBy: 'translations')] #[ORM\JoinColumn(nullable: false, onDelete: 'CASCADE')] private Event $event, #[ORM\Column(type: Types::STRING, length: 5, nullable: false)] private string $locale)
```

Comment:
this was inlined by rector automatically. i think it would be easier to read otherwise, but i am not sure if i can ignore the rule just in this case yet. will check this once the other issues are resolved

Reply:
adding line breaks should make it readable ;) 

```suggestion
    public function __construct(
        #[ORM\ManyToOne(targetEntity: Event::class, inversedBy: 'translations')]
        #[ORM\JoinColumn(nullable: false, onDelete: 'CASCADE')]
        private Event $event,
        #[ORM\Column(type: Types::STRING, length: 5, nullable: false)]
        private string $locale)
```

PS: rector has no code style and if php cs fixer has no rule yet it will stay as one thing.